[Documentation] Updating main README with new install markdowns (#6712)
aces/Loris: README.md

* Try the LORIS demo instance at https://demo.loris.ca.

This Readme covers installation of LORIS version <b>23</b> on <b>Ubuntu</b>.
([CentOS Readme also available](docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md)).

> Please consult the [Ubuntu Installation guide](docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md) or [CentOS Installation guide](docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md) for more information. Installation instructions and further information about LORIS for developers can be found on the [LORIS ReadTheDocs website](https://acesloris.readthedocs.io/en/latest/).

Reviewer: ```suggestion
Please consult the [Ubuntu Installation guide](docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md) or [CentOS Installation guide](docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md) for more information. These installation instructions and more LORIS documentation for developers can also be found on the [LORIS ReadTheDocs website](https://acesloris.readthedocs.io/en/latest/).
```
my comment was not addressed.

This Readme covers installation of LORIS version <b>23</b> on <b>Ubuntu</b>.
([CentOS Readme also available](/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md)).

> Please consult the [LORIS Installation Markdowns](/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install) for more information. Installation instructions and further information about LORIS for developers can be found on the [LORIS ReadTheDocs website](https://acesloris.readthedocs.io/en/latest/).

Author: Should I include something along the lines of:

> Please consult the LORIS Installation Markdowns **located in the /docs/wiki directory**.

Reviewer: if we can avoid the confusing path `docs/wiki/` that would be nice, to minimize changes down the road

Reviewer: note: [alex's comment below](https://github.com/aces/Loris/pull/6712#issuecomment-642218561) was in response to this thread

> Please consult the [LORIS Installation Markdowns](/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install) for more information. Installation instructions and further information about LORIS for developers can be found on the [LORIS ReadTheDocs website](https://acesloris.readthedocs.io/en/latest/).

Reviewer: The ReadTheDocs is just a web-friendly presentation of markdowns, and the Installation Instructions you mention there _are_ these Installation markdowns, no?  
This line suggests the RTD docs don't include the same install instruction content.

Author: The readthedocs does include the installation instructions but it also includes all of the documentation available under docs/wiki. Would it be better to just link to readthedocs and not link to `/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install` at all?

Reviewer 2: Absolute links will break readthedocs, and readthedocs can't manager link to folders. It's fine if we don't want to include the main README there though.

Reviewer: i'll leave the points about which links break to laetitia.  keep all the mentions, and just make clear the subset relationship between them - i.e. that this same content is _also_ readable on ReadTheDocs, which also includes more docs...

> Please consult the [LORIS Installation Markdowns](/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install) for more information. Installation instructions and further information about LORIS for developers can be found on the [LORIS ReadTheDocs website](https://acesloris.readthedocs.io/en/latest/).

Reviewer 2: `Please consult the [Ubuntu Installation guide](docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md) or [CentOS Installation guide](docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md) for more information. Installation instructions and further information about LORIS for developers can be found on the [LORIS ReadTheDocs website](https://acesloris.readthedocs.io/en/latest/).`

Reviewer 2: This should be compatible with readthedocs if we want the main README to be part of it in the future.

Author: Ok I can do it this way. Sorry I was confused about what you meant in previous comments. Thanks :)

### Install Steps

> Consult the [LORIS Installation Markdowns](/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install) for more information.

Reviewer 2: Same here: Absolute links will break readthedocs, and readthedocs can't manage links to folders.

### Install Steps

> <<<<<<< HEAD

Reviewer 3: There's a merge conflict marker here

Reviewer: @alex this is resolved right? 👍

3. Run installer script to install core code, and libraries. The script will prompt for information and so that it can create directories automatically.

> For more information, please read the [Installation Markdowns](/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install).

Reviewer 2: And again :)

## Community

### Get in touch
For questions and troubleshooting guidance beyond what is covered in our documentation, please subscribe to the [LORIS Developers mailing list](http://www.bic.mni.mcgill.ca/mailman/listinfo/loris-dev) and email us there. 

### Support and GitHub Issues

> For troubleshooting specific installation issues or errors, please see the [Installation troubleshooting guide](), and then contact us via the [LORIS Developers mailing list](http://www.bic.mni.mcgill.ca/mailman/listinfo/loris-dev).

Reviewer: ```suggestion
For troubleshooting specific installation issues or errors, please see the [Installation troubleshooting guide](), and then contact us via the [LORIS Developers mailing list](http://www.bic.mni.mcgill.ca/mailman/listinfo/loris-dev).
```
@AlexandraLivadas could you add the link in here, and read over this commit (I fixed the last orphan wiki reference for ya.)
